From 49193910e0fa1476469e4458852a2f17861e1d4b Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Wed, 25 Oct 2017 17:19:41 -0600
Subject: [PATCH 026/249] wiki: Clarify which klte* build is for which model

* Listing the builds by country and/or carrier when the difference is
  based on device model is not intuitive to users. Give an indication
  which build is for which model

Change-Id: I6b46514a81627eb2cd31c06a321352b030f2ac1a
---
 _data/devices/klte.yml       | 2 +-
 _data/devices/kltechn.yml    | 2 +-
 _data/devices/kltechnduo.yml | 2 +-
 _data/devices/klteduos.yml   | 2 +-
 _data/devices/kltedv.yml     | 2 +-
 _data/devices/kltekdi.yml    | 2 +-
 _data/devices/kltekor.yml    | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/_data/devices/klte.yml b/_data/devices/klte.yml
index 8a7731c..aaf41b7 100644
--- a/_data/devices/klte.yml
+++ b/_data/devices/klte.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G900AZ, SM-G900F, SM-G900M, SM-G900R4, SM-G900R7, SM-G900T, SM-G900V, SM-G900W8, SM-S902L]
-name: Galaxy S5
+name: Galaxy S5 LTE (G900AZ/F/M/R4/R7/T/V/W8,S902L)
 network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM, 800 1900MHz 1xRTT (G900R4/G900R7/G900V/S902L)' }
 - {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA (G900F/G900V), 850 900 1700 1900 2100 MHz HSDPA (G900M), 850 1700 1900 2100 MHz HSDPA (G900T), 800 1900MHz EVDO (G900R4/G900R7/G900V/S902L)' }
diff --git a/_data/devices/kltechn.yml b/_data/devices/kltechn.yml
index efaa900..10911fa 100644
--- a/_data/devices/kltechn.yml
+++ b/_data/devices/kltechn.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G9006V, SM-G9008V]
-name: Galaxy S5 (China)
+name: Galaxy S5 LTE (G9006V/8V)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
diff --git a/_data/devices/kltechnduo.yml b/_data/devices/kltechnduo.yml
index 8fd2706..f1115e6 100644
--- a/_data/devices/kltechnduo.yml
+++ b/_data/devices/kltechnduo.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G9006W, SM-G9008W, SM-G9009W]
-name: Galaxy S5 Duos (China)
+name: Galaxy S5 LTE Duos (G9006W/8W)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
diff --git a/_data/devices/klteduos.yml b/_data/devices/klteduos.yml
index c2c829b..af20207 100644
--- a/_data/devices/klteduos.yml
+++ b/_data/devices/klteduos.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G900FD, SM-G900MD]
-name: Galaxy S5 Duos
+name: Galaxy S5 LTE Duos (G900FD/MD)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
diff --git a/_data/devices/kltedv.yml b/_data/devices/kltedv.yml
index 5401818..9624fc1 100644
--- a/_data/devices/kltedv.yml
+++ b/_data/devices/kltedv.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G900I, SM-G900P]
-name: Galaxy S5 (G900I/P)
+name: Galaxy S5 LTE (G900I/P)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
diff --git a/_data/devices/kltekdi.yml b/_data/devices/kltekdi.yml
index ab26d99..5a4a0f8 100644
--- a/_data/devices/kltekdi.yml
+++ b/_data/devices/kltekdi.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SCL23]
-name: Galaxy S5 (AU)
+name: Galaxy S5 LTE (SCL23)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
diff --git a/_data/devices/kltekor.yml b/_data/devices/kltekor.yml
index b606330..562e427 100644
--- a/_data/devices/kltekor.yml
+++ b/_data/devices/kltekor.yml
@@ -19,7 +19,7 @@ install_method: heimdall
 kernel: android_kernel_samsung_msm8974
 maintainers: [haggertk, javelinanddart]
 models: [SM-G900K, SM-G900L, SM-G900S]
-name: Galaxy S5 (Korea)
+name: Galaxy S5 LTE (G900K/L/S)
 network:
 - {tech: 2G, bands: '' }
 - {tech: 3G, bands: '' }
-- 
2.17.1

